### PR TITLE
Develop (challenge + jamine)

### PR DESCRIPTION
**Pull request by doboapps**

### CODE REVIEW

File: staff/doboapps/js/challenges/scripts/find.js [hunk 1]
```diff
@@ -19,7 +19,9 @@
  * 
  * 
  */
+var vac = $("vacations")
```
Comment (Asix94) [inline staff/doboapps/js/challenges/scripts/find.js]: WHY ?? CLEAN CODE

Comment (doboapps) [reply staff/doboapps/js/challenges/scripts/find.js]: oops! 😄

Comment (doboapps) [reply staff/doboapps/js/challenges/scripts/find.js]: It is done! 😉

**Review by agandia9 — COMMENTED** [review]
OK

**Pull request merged**